chainerx_cc/chainerx/third_party/abseil.cmake

```diff
@@ -0,0 +1,14 @@
+cmake_minimum_required(VERSION 3.1)
+project(abseil-download NONE)
+
+include(ExternalProject)
+ExternalProject_Add(pybind11
+    GIT_REPOSITORY    https://github.com/abseil/abseil-cpp.git
+    GIT_TAG           72e09a54d993b192db32be14c65adf7e9bd08c31
```

Comment:
In principal we should not rely on a pre-release commit.
How about adding a TODO?